create_auction.php

```diff
@@ -28,37 +33,65 @@
         <div class="form-group row">
           <label for="auctionTitle" class="col-sm-2 col-form-label text-right">Title of auction</label>
           <div class="col-sm-10">
-            <input type="text" class="form-control" id="auctionTitle" placeholder="e.g. Black mountain bike">
+            <input type="text" class="form-control" id="auctionTitle" name="auctionTitle" placeholder="e.g. Black mountain bike">
             <small id="titleHelp" class="form-text text-muted"><span class="text-danger">* Required.</span> A short description of the item you're selling, which will display in listings.</small>
           </div>
         </div>
+        <div class="form-group row">
+          <label class="col-sm-2 col-form-label text-right">Select Items</label>
+          <div class="col-sm-10">
+              <?php
+              //$seller_id = "5";
+              // Retrieve seller's items
+              $seller_items = getSellerItems($seller_id);
+              // Display checkboxes for each item
+              foreach ($seller_items as $item) {
+                  echo '<div class="form-check">';
+                  echo '<input class="form-check-input" type="checkbox" name="selectedItems[]" value="' . $item['id'] . '">';
+                  echo '<label class="form-check-label">' . $item['name'] . '</label>';
+                  echo '</div>';
+              }
+              ?>
+          </div>
+        </div>
         <div class="form-group row">
           <label for="auctionDetails" class="col-sm-2 col-form-label text-right">Details</label>
           <div class="col-sm-10">
-            <textarea class="form-control" id="auctionDetails" rows="4"></textarea>
+            <textarea class="form-control" id="auctionDetails" name="auctionDetails" rows="4"></textarea>
             <small id="detailsHelp" class="form-text text-muted">Full details of the listing to help bidders decide if it's what they're looking for.</small>
           </div>
         </div>
-        <div class="form-group row">
+        <!-- <div class="form-group row">
           <label for="auctionCategory" class="col-sm-2 col-form-label text-right">Category</label>
           <div class="col-sm-10">
-            <select class="form-control" id="auctionCategory">
+            // Load categories dynamically from the database 
+            <?php
+            $categories = getCategoriesFromDatabase(); // Implement a function to fetch categories
+            ?>
+            <select class="form-control" id="auctionCategory" name="auctionCategory">
               <option selected>Choose...</option>
-              <option value="fill">Fill me in</option>
-              <option value="with">with options</option>
-              <option value="populated">populated from a database?</option>
+              <?php foreach ($categories as $category) : ?>
+                <option value="<?php echo $category['id']; ?>"><?php echo $category['name']; ?></option>

```

Comment:
Do we need categories in Auction creation now that we are having multiple Items in an auction?